guideline/cfh.md

```diff
@@ -131,26 +131,11 @@ off by the PAC before they can be put in place.
    Proposals will have to provide details about how these can be implemented at
    the proposed venues.
 
-   1. Food and drinks for lunches and breaks on all conference days (and
-      tutorials and sprints where applicable)
-
-   2. Optionally, breakfast on all conference days (not necessarily on the
-      tutorial and sprint days where applicable)
-
    3. WLAN service to access the Internet during the conference (and the
       tutorial and sprints if applicable)
 
-   4. At least one social event, which can be a dinner, a show or some other
```

Comment:
I am OK if there isn't one in a PyCon APAC, but lacking a social event will reduce my intention to attend it.  Having this item in the plan requirement reminds proposers its benefits.  But since it's good to have, I am OK if we remove this item.